pkg/prometheus/server.go

```diff
@@ -22,6 +24,8 @@ import (
 	"github.com/prometheus/prometheus/prompb"
 )
 
+const GRACEFUL_TIMEOUT_SECONDS int = 5
```

Comment:
may be lets increase this to 15-30 seconds ?
sending each 1000 metrics can take up to 4 seconds

Reply:
its timeout for http server to stop accept new metrics, the timeout for metrics flush and push will be limited outside of service.